cmd/restore_repo.go

```diff
@@ -15,6 +15,17 @@ import (
 	"github.com/urfave/cli"
 )
 
+var defaultUnits = cli.StringSlice{
+	"wiki",
+	"issues",
+	"labels",
+	"releases",
+	"release_assets",
+	"milestones",
+	"pull_requests",
+	"comments",
+}
+
```

Comment:
I think this default value is not ideal for the server code.

The default value should be 0-length slice to indicate all units (there may be more in the future).

![image](https://user-images.githubusercontent.com/2114189/173217740-4af9bf59-4ff7-4df4-a9d3-0e0a87d5601e.png)

Reply:
I think it's a good idea to reduce the possibility to forget the places when we support more units.